### DISCUSSION TITLE
Wrote a review. Plausible Analytics: 10 Reasons it's the Best Web Analytics Tool

### DISCUSSION THREAD

**Discussion by derpycoder** [plausible/analytics#799] [Anything Else]
I wrote a blog post after becoming a paid member.

Here it is:

[Plausible Analytics: 10 Reasons it's the Best Web Analytics Tool](https://www.derpycoder.com/plausible-analytics-10-reasons-its-the-best-web-analytics-tool/?utm_campaign=Plausible%20Analytics&utm_source=Plausible&utm_medium=Forum)

**Comment by metmarkosaric** [comment]
thanks very much @abhijit-kar! looking great! [i've added it to our docs now](https://plausible.io/docs/plausible-analytics-reviews).

and i'll see if there's a way to highlight keyboard shortcuts on our homepage somehow
